src/styles/_catalog.scss

```diff
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(2, 200px);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
```

Comment:
I think you need to remove `display: grid` from the card, because it makes the card the grid of its own, which is not necessary here. The card is an element of the `.catalog` grid, but not a grid of its own.